.github/workflows/ci.yml

```diff
@@ -125,11 +125,67 @@ jobs:
       - name: Install FuseML
         run: |
           ./dist/fuseml-installer version
-          make fuseml-install-with-extensions
+          make fuseml-install-with-kfserving
 
-      - name: Run mlflow-e2e
+      - name: Run mlflow-e2e with kfserving
         run: |
            # extract the target branch in the context of which this job is running
            branch=${{ github.base_ref }}
            [[ -z "$branch" ]] && branch=${GITHUB_REF#refs/heads/}
-           RELEASE_BRANCH=$branch make mlflow-e2e
+           RELEASE_BRANCH=$branch make mlflow-kfserving-e2e
+
+  mlflow-seldon-e2e:
```

Comment:
instead of duplicating all that code, you could use: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
done